website/site/content/docs/customization.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Custom Previews
```

Comment:
We should change the form of the titles to be the same: "Creating Custom Previews" would match "Creating Custom Widgets".

Reply:
Good point.